src/Components/Web/src/Forms/InputFile/BrowserFileStream.cs

```diff
@@ -67,6 +87,39 @@ public override async ValueTask<int> ReadAsync(Memory<byte> buffer, Cancellation
             return bytesRead;
         }
 
-        protected abstract ValueTask<int> CopyFileDataIntoBuffer(long sourceOffset, Memory<byte> destination, CancellationToken cancellationToken);
+        private async Task<Stream> OpenReadStreamAsync(RemoteBrowserFileStreamOptions options, CancellationToken cancellationToken)
+        {
+            var dataReference = await _jsRuntime.InvokeAsync<IJSStreamReference>(
+                InputFileInterop.ReadFileData,
+                cancellationToken,
+                _inputFileElement,
+                _file.Id);
+
+            return await dataReference.OpenReadStreamAsync(
+                _maxAllowedSize,
+                cancellationToken: cancellationToken);
+        }
+
+        private async ValueTask<int> CopyFileDataIntoBuffer(long sourceOffset, Memory<byte> destination, CancellationToken cancellationToken)
```

Comment:
Is this still used? Who calls it? I see it was abstract before but can't work out in what way it would still be used.

Reply:
Never mind, I see where it's called now.